sdk/spring/scripts/compatibility_insert_dependencymanagement.py

```diff
@@ -120,12 +108,45 @@ def add_dependency_management_for_file(file_path, spring_boot_dependencies_versi
             repo_content = pom_file_content[:insert_position] + insert_content + pom_file_content[insert_position:]
             with open(file_path, 'r+', encoding = 'utf-8') as updated_pom_file:
                 updated_pom_file.writelines(repo_content)
+    with open(file_path, 'r', encoding = 'utf-8') as pom_file:
+        pom_file_content = pom_file.read()
+        insert_position = get_dependency_management_position(pom_file_content)
+        insert_content = get_dependency_management_content(pom_file_content)
+        dependency_content = pom_file_content[:insert_position] + insert_content + pom_file_content[insert_position:]
```

Comment:
how about using a variable here, `updated_pom_file_content` instead of `dependency_content` and `prop_content`?